drivers/fuel_gauge/sbs_gauge/emul_sbs_gauge.c

```diff
@@ -367,7 +367,7 @@ static int emul_sbs_sbs_gauge_init(const struct emul *target, const struct devic
 	static struct sbs_gauge_emul_data sbs_gauge_emul_data_##n;                                 \
 	static const struct sbs_gauge_emul_cfg sbs_gauge_emul_cfg_##n = {                          \
 		.addr = DT_INST_REG_ADDR(n),                                                       \
-		.cutoff_support = DT_PROP_OR(DT_DRV_INST(n), battery_cutoff_support, false),       \
+		.cutoff_support = DT_PROP(DT_DRV_INST(n), battery_cutoff_support),       \
```

Comment:
the binding doesnt have `battery_cutoff_support` prop

Reply:
Sorry was sick last week. Looking today.